### DISCUSSION TITLE
Update of Staging Instance URLs

### DISCUSSION THREAD

**Discussion by jenriquesoriano** [INSPIRE-MIF/helpdesk-validator#1105] [Announcements]
Dear Community,

We are pleased to announce that the URLs for the Staging instance of the INSPIRE Reference Validator have been updated to the inspire.europa.ec.eu domain.

From now on, you can access the Staging instance and its API through the following links:
- Staging Instance: https://inspire.ec.europa.eu/validator-staging/home/index.html
- Staging API: https://inspire.ec.europa.eu/validator-staging-api/

Please note that the old link will no longer be available, so we kindly ask you to update your bookmarks and references to the new URLs mentioned above.

Best regards,

**Comment by HelmoetOrdina** [comment]
Dear Mr. Soriano,

Thanks for the update.

Please note that the staging API url from your mail below seems to work improperly (Figure 1 and 2):

Maybe I’m mistaking?

Regards, Helmoet de Zwijger.

***@***.***

Figure 1
***@***.***

Figure 2

Van: jenriquesoriano ***@***.***>
Verzonden: vrijdag 9 augustus 2024 14:38
Aan: INSPIRE-MIF/helpdesk-validator ***@***.***>
CC: Subscribed ***@***.***>
Onderwerp: [INSPIRE-MIF/helpdesk-validator] Update of Staging Instance URLs (Discussion #1105)


Dear Community,

We are pleased to announce that the URLs for the Staging instance of the INSPIRE Reference Validator have been updated to the inspire.europa.ec.eu domain.

From now on, you can access the Staging instance and its API through the following links:

  *   Staging Instance: https://inspire.ec.europa.eu/validator-staging/home/index.html
  *   Staging API: https://inspire.ec.europa.eu/validator-staging-api/

Please note that the old link will no longer be available, so we kindly ask you to update your bookmarks and references to the new URLs mentioned above.

Best regards,

—
Reply to this email directly, view it on GitHub<https://github.com/INSPIRE-MIF/helpdesk-validator/discussions/1105>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A3OQ324YEIQQCQVMDBON76TZQSZYZAVCNFSM6AAAAABMIMQIQCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGAZDSNZSHE>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.******@***.***>>

Disclaimer
Dit bericht met eventuele bijlagen is vertrouwelijk en uitsluitend bestemd voor de geadresseerde. Indien u niet de bedoelde ontvanger bent, wordt u verzocht de afzender te waarschuwen en dit bericht met eventuele bijlagen direct te verwijderen en/of te vernietigen. Het is niet toegestaan dit bericht en eventuele bijlagen te vermenigvuldigen, door te sturen, openbaar te maken, op te slaan of op andere wijze te gebruiken. Ordina Holding B.V. en/of haar groepsmaatschappijen accepteren geen verantwoordelijkheid of aansprakelijkheid voor schade die voortvloeit uit de inhoud en/of de verzending van dit bericht.

This e-mail and any attachments are confidential and are solely intended for the addressee. If you are not the intended recipient, please notify the sender and delete and/or destroy this message and any attachments immediately. It is prohibited to copy, to distribute, to disclose or to use this e-mail and any attachments in any other way. Ordina Holding B.V. and/or its group companies do not accept any responsibility nor liability for any damage resulting from the content of and/or the transmission of this message.

**Reply by jenriquesoriano** [reply]
Dear @HelmoetOrdina ,

thanks for your feedback, we are afraid that we can't see the attached figures.
In any case, please let us clarify in case we can help you to solve the issue.

The URL for Staging API has been updated from:
http://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/validator/v2/
to:
https://inspire.ec.europa.eu/validator-staging-api/

As a result, calls to API methods such as:
http://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/validator/v2/status
will no longer work. 
Instead, you should use the new URL:
https://inspire.ec.europa.eu/validator-staging-api/status

Thus, in case you were using the Staging API this replacement should be made in the methods you are using.

Please let us know if this feedback helps.

Best regards,

**Comment by HelmoetOrdina** [comment]
I generally don't use the staging version of the validation service, I was checking the urls and didn't get the new staging api working. I will check again using your (this) feedback next week.
Thanks anyway.

Verzonden vanaf Outlook voor Android<https://aka.ms/AAb9ysg>
________________________________
